### PR TITLE
[DEV APPROVED] - 7458 automated fca import

### PR DESCRIPTION
**Pull request by jriga**
**Issue**
The weekly FCA data import needs some improvement. 
More details on [Ticket 7458](https://moneyadviceservice.tpondemand.com/entity/7458)

**Proposed Solution**
This solution uses Azure to storage file online.

- [x] 1. Find app for user to upload file with ease on azure
- [x] 2. Admin UI for listing files ready for import
- [x] 3. Create documentation for user
- [x] 4. Seperate log file 
- [x] 5. Perform data import
- [x] 6. Notify use of import outcome
- [x] 7. Confirm data import

### CODE REVIEW

**Comment by tomas-stefano** [comment]
👍 👍

File: app/assets/javascripts/admin/lookup/fca_import.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (jongilbraith) [inline app/assets/javascripts/admin/lookup/fca_import.js]: Can remove this.

**Comment by jongilbraith** [comment]
Pretty epic piece of work!

Nice test coverage.

👍

File: app/assets/javascripts/admin/lookup/fca_import.js (hunk 1 above)
Comment (jriga) [inline app/assets/javascripts/admin/lookup/fca_import.js]: ♻️

**Pull request merged**